File: All-Services/ClientApp/src/ViewModels/Company/searchCompanyViewModel.ts
export class SearchCompanyViewModel {
  constructor(
    public companyId?: number,
    public mame?: string,
    public desc?: string,
    public logoUrl?: string,
    public webSiteUrl?: string,
    public telephone?: string,
    public likes?: number,
    public disLikes?: number
  ) { }
}

File: All-Services/ViewModels/Company/SearchCompanyViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace All_Services.ViewModels.Company
{
    public class SearchCompanyViewModel
    {
        public int CompanyId { get; set; }
        public string Name { get; set; }
        public string Desc { get; set; }
        public string LogoUrl { get; set; }
        public string WebSiteUrl { get; set; }
        public string Telephone { get; set; }
        public int Likes { get; set; }
        public int DisLikes { get; set; }
    }
}
